packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/add-node/translations/Messages_en_GB.json

```diff
@@ -1,8 +1,8 @@
 {
-  "pci_databases_general_information_add_node": "Add node",
+  "pci_databases_general_information_add_node": "Add a node",
   "pci_databases_general_information_add_node_confirm": "Confirm",
   "pci_databases_general_information_add_node_cancel": "Cancel",
-  "pci_databases_general_information_add_node_description": "You currently have {{nodesCount}} nodes. You can add a node for",
+  "pci_databases_general_information_add_node_description": "You currently have {{nodesCount}} nodes. Do you really want to add a node ? Your billing will be increased by",
```

Comment:
You have only updated english text. What about other languages? The right way is to update source language(french) and ask for translation.

Reply:
I have updated the translations, thanks ! https://github.com/ovh/manager/commit/0eb2da847ec72f1ba48b9866988490efec66e013